coalib/bears/PackageRequirement.py

```diff
@@ -0,0 +1,37 @@
+class PackageRequirement:
+
+    def __init__(self, manager: str, package: str, version):
+        """
+        :param manager: A string with the name of the manager (pip, npm, etc).
+        :param package: A string with the name of the package to be installed.
+        :param version: A number that contains the version.
+        """
+        self.manager = manager
+        self.package = package
+        self.version = version
+
+    def __str__(self):
+        return "{.package} version {.version} (for {.manager})".format(self)
+
+    def check(self):
+        """
+        Check if the requirement is satisfied.
+        """
+        raise NotImplementedError
+
+
+class PythonRequirement(PackageRequirement):
+
+    def __init__(self, package: str, version):
+        """
+        :param package: A string with the name of the package to be installed.
+        :param version: A number that contains the version.
+        """
+        self.package = package
+        self.version = version
```

Comment:
nonononono! You have to invoke `PackageRequirement(package, 'pip', version)` to do the parental construction